stimuli-ci-generation/Dockerfile_base

```diff
@@ -0,0 +1,21 @@
+FROM si_rocker_1
+
+RUN apt-get update 
+
+
+RUN apt-get install -y --no-install-recommends python3.7 
+
+RUN apt-get install -y --no-install-recommends python3-pip 
+
+RUN apt-get install -y --no-install-recommends python3-setuptools 
+
+RUN apt-get install -y --no-install-recommends python3-dev
```

Comment:
these can be collapsed into one install line

Reply:
I prefer to keeping it this way, looks clearer what's needed and easier to modify in the future